docs/source/community/host-guide.md

```diff
@@ -16,113 +12,150 @@ Each meeting should have a collaborative public agenda document (for example on
 
 ## &#x1F3AF; Meeting Purpose/Goals
 
-Meetings are a way to connect members of our 
-community. All participants should feel welcome and be heard. In line with that:
+Meetings are a way to connect members of our community. All participants should feel welcome and be heard. In line with that:
 
-Meetings should be...
+Meetings should be…
 
-- Positive and friendly
-- Welcoming and open to people from diverse backgrounds
-- Newcomers in particular should be welcomed and encouraged to
-  chime in, discuss, participate, or to simply observe if that is their preference.
+* Positive and friendly  
+* Welcoming and open to people from diverse backgrounds  
+* Newcomers in particular should be welcomed and encouraged to chime in, discuss, participate, or to simply observe if that is their preference.
 
 ## &#x1F4AC; Topics
 
 Each meeting should have a list of topics that are generally discussed. Contributors can propose topics before or during the meeting.
 
 (If using this document as a template, replace this section with a list of topics that are discussed at your meeting.)
 
+## **Before the Meeting**
+
+* Select date (at least 4 weeks in advance) and time  
+  * In the past we’ve taken over JupyterLab weekly time slot. This encourages participation from subproject that is accustomed to meeting during a particular time. [Link to template email](https://docs.google.com/document/d/1GJjHs21XcmT67755Hu6ryR0FGE_5JQYruaNjBRdqR4c/edit?tab=t.yxduni8mhh1z).  
+* Create HackMD  
+  * Create HackMD account   
+  * Start a new note  
+  * Use template [here](https://drive.google.com/file/d/1IQ5T7VJCI1dKk_CMXDjbtrKG8W_uf4E2/view?usp=sharing)  
```

Comment:
Same comment here, I think we probably for this one would want to include it in this repo and link to that?